app/views/responses/show.html.haml

```diff
@@ -1,8 +1,30 @@
-%tr
-	%td= link_to "Edit", edit_response_path(@response)
-	%td
-		= form_for @response, url: response_path(@response.id), html: {:onsubmit => "return confirm('Are you sure you want to delete the question?');"}, method: :delete do |f|
-			= submit_tag('Delete')
-%p= @response.text
-%p= @response.user_id
-%p= (@response.created_at).strftime("%m-%d-%Y")
\ No newline at end of file
+%html{:lang => "en"}
```

Comment:
remove the html, body and head tags. They are rendered in the surrounding layout and shouldn't be inserted in the view.